<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>501</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="VTPLabel" name="labelUser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="VTPLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushBtnLogin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUser">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameData">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushBtnSelectFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>120</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushBtnSendData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>120</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSelectedFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameResult">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListView" name="listViewResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VTPLabel</class>
   <extends>QLabel</extends>
   <header location="global">VTPLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
